frontend/src/lib/components/Support/supportLogic.ts

```diff
@@ -193,7 +239,7 @@ export const supportLogic = kea<supportLogicType>([
                         zendesk_ticket_id,
                         zendesk_ticket_link: `https://posthoghelp.zendesk.com/agent/tickets/${zendesk_ticket_id}`,
                     }
-                    posthog.capture('support_ticket', properties)
+                    actions.reportSupportFormSubmitted(properties)
```

Comment:
TBH I think `eventUsageLogic` is now kind of an anti-pattern. It was concocted by Paolo a long time ago, but since then its centralization hasn't been super useful, it just introduces an extra layer of code